src/main/java/org/skriptlang/skriptworldguard/elements/expressions/ExprRegionPriority.java

```diff
@@ -0,0 +1,101 @@
+package org.skriptlang.skriptworldguard.elements.expressions;
+
+
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+
+import com.sk89q.worldguard.protection.regions.ProtectedRegion;
+
+import org.bukkit.event.Event;
+import org.skriptlang.skriptworldguard.SkriptWorldGuard;
+import org.skriptlang.skriptworldguard.worldguard.WorldGuardRegion;
+
+public class ExprRegionPriority extends SimplePropertyExpression<WorldGuardRegion, Number> {
+
+    static {
+
+        register(ExprRegionPriority.class, Number.class, "priority", "worldguardregions");
+
+    }
+
+
+    private Expression<WorldGuardRegion> region;
+
+    @SuppressWarnings("unchecked")
+    @Override
```

Comment:
[Code Conventions](https://github.com/SkriptLang/Skript/blob/master/code-conventions.md)
```suggestion
    @Override
    @SuppressWarnings("unchecked")
```